svo (curing skeleton, controllers, action system).xml

```diff
@@ -3026,6 +3040,7 @@ end
 svo.savesettings = svo.QQ
 
 -- add in blackout only, otherwise go off the prompt - this allows for time tracking
+--maybe can remove this now if it was only for tracking eq in blackout?
```

Comment:
This was to protect against illusions that would make you think that you are off balance, or disrupt in blackout where you'd come out with no eq and would never regain it. If it's not a thing anymore in the game - then go for it